api/desecapi/tests/testregistration.py

```diff
@@ -105,20 +108,23 @@ def test_multiple_registration_not_locked_same_ip_long_time(self):
     def test_send_captcha_email_manually(self):
         outboxlen = len(mail.outbox)
 
-        url = reverse('register')
+        url = reverse('v1:register')
         data = {'email': utils.generateUsername(),
                 'password': utils.generateRandomString(size=12), 'dyn': True}
         response = self.client.post(url, data, REMOTE_ADDR="1.3.3.10")
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         user = models.User.objects.get(email=data['email'])
-        send_account_lock_email(None, user)
+        r = RequestFactory().request(HTTP_HOST=settings.ALLOWED_HOSTS[0])
+        r.version = 'v1'
+        r.versioning_scheme = NamespaceVersioning()
```

Comment:
See my comment in `urls.py`. Knowledge of the versioning scheme class is overspecific here, I think. A wrapper would solve this.

Reply:
It is very specific, and it needs to be, as the framework has no other source of knowing which versioning scheme is used. Since we're testing if there is an email sent, it's also not important which scheme we're using.

It's the only place where we need to provide a mocked request, which is why I do not want to wrap it.

Reply:
The code _can_ figure out which versioning scheme is used, by looking at the settings. Anyways, I agree with your conclusion.